Hazelcast.Net/Hazelcast.Client.Network/Connection.cs

```diff
@@ -75,31 +75,30 @@ internal sealed class Connection
         public Exception CloseCause { get; private set; }
 
         public Connection(ConnectionManager connectionManager, InvocationService invocationService, int id, Address address,
-            ClientNetworkConfig clientNetworkConfig)
+            NetworkConfig clientNetworkConfig)
         {
             _connectionManager = connectionManager;
             _id = id;
 
             var isa = address.GetInetSocketAddress();
-            var socketOptions = clientNetworkConfig.GetSocketOptions();
+            var socketOptions = clientNetworkConfig.SocketOptions;
             try
             {
                 _clientSocket = new Socket(isa.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
                 _clientSocket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.KeepAlive, true);
-                if (socketOptions.GetLingerSeconds() > 0)
+                if (socketOptions.LingerSeconds > 0)
                 {
-                    var lingerOption = new LingerOption(true, socketOptions.GetLingerSeconds());
+                    var lingerOption = new LingerOption(true, socketOptions.LingerSeconds);
                     _clientSocket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.Linger, lingerOption);
                 }
                 else
                 {
                     var lingerOption = new LingerOption(true, 0);
                     _clientSocket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.DontLinger, lingerOption);
                 }
-                _clientSocket.NoDelay = socketOptions.IsTcpNoDelay();
-                _clientSocket.ReceiveTimeout = socketOptions.GetTimeout() > 0 ? socketOptions.GetTimeout() : -1;
```

Comment:
So I think you answered already, but just wanted to make sure removing `ReceiveTimeout` is intentional - I'm ok with it.